examples/step-63/doc/results.dox

```diff
@@ -1,2 +1,440 @@
+</tr>
+</table>
+
+We see that for $Q_1$, both multiplicative smoothers require a smaller combination of smoothing steps and iteration counts than either additive smoother. However, when we in increase the degree to a $Q_3$ element, there is a clear advantage for the block smoothers in terms of the increase in smoothing steps and iterations required to solve. Specifically, the block SOR smoother gives constant iterations over degree, and the block Jacobi only sees about a 38% increase in iterations compared to 75% and 183% for Jacobi and SOR respectively.
+
+<h3> Cost </h3>
+
+Iteration counts do not tell the full story in the optimality of a one smoother over another. Obviously we must examine the cost of an iteration. Block smoothers here are at a disadvantage as they are having to construct and invert a cell matrix for each cell. Here is a comparison of solve times for a $Q_3$ element with 74,496 DoFs:
+ 
+<table align="center" border="1">
+<tr>
+  <th colspan="1">$Q_3$</th>
+  <th colspan="4">Smoother (smoothing steps)</th>
+</tr>
+<tr>
+  <th colspan="1">DoFs</th>
+  <th colspan="1">Jacobi (6)</th>
+  <th colspan="1">Block Jacobi (3)</th>
+  <th colspan="1">SOR (3)</th>
+  <th colspan="1">Block SOR (1)</th>
+</tr>
+<tr>
+  <th>74496</th>
+  <td>0.689 s</th>
+  <td>5.826 s</th>
+  <td>1.189 s</th>
+  <td>1.021 s</th>
+</tr>
+</table>
+
+The smoother that requires the most iterations (Jacobi) actually take the shortest time (roughly 2/3 the time of the next fastest method). This is because all that is requires to apply a Jacobi smoothing step is multiplication by a diagonal matrix which is very cheap. On the other hand, while SOR requires over 3x more iterations (each with 3x more smoothing steps) than block SOR, the times are roughly equivalent, implying that a smoothing step of block SOR is roughly 9x slower than a smoothing step of SOR. Lastly, block Jacobi is almost 6x more expensive than block SOR, which intuitively makes sense from the fact that 1 step of each method has the same cost (inverting the cell matrices and either adding or multiply them together), and block Jacobi has 3 times the smoothing step per iteration with 2 times the iterations.
+
+
+<h3> Additional Points </h3>
+
+There are a few more important points to mention:
+
+1. For a mesh distributed in parallel, multiplicative methods cannot be executed over the entire domain. One can use a hybrid method where a multiplicative smoother is applied on each subdomain, but as you increase the number of subdomains, the method converges to that of an additive method. This is a major disadvantage to these methods.
+
+2. Current research into block smoothers suggest that soon we will be able to compute the inverse of the cell matrices much cheaper than what is currently being done inside deal.II. It seems that one should be able to take advantage of matrix free implementations and also the fact that, in the interior of the domain, cell matrices tend to look very similar. 
```

Comment:
You could mention that this research is based on an old idea (the fast diagonalization method, a concept from the 1960s), and it has been used in the spectral element community for around 20 years (there are papers e.g. by Paul Fischer and others, see e.g. https://doi.org/10.1007/s10915-004-4787-3 ) but there are currently efforts to generalize them (to DG) and to make them more robust.